epitome/test/dataset_test.py

```diff
@@ -205,3 +220,40 @@ def test_reserve_validation_indices(self):
         joined_indices = np.concatenate([dataset.indices[Dataset.TRAIN_VALID], dataset.indices[Dataset.TRAIN]])
         joined_indices.sort()
         self.assertTrue(len(np.setdiff1d(joined_indices, old_indices)) == 0 and len(np.setdiff1d(old_indices, joined_indices)) == 0)
+
+    def test_get_data_dir(self):
+        # Test unspecified model to have default data dir path
+        assert self.dataset.data_dir == self.epitome_test_dir
+
+        # Create new dataset with new undownloaded data path
+        epitome_test_data_dir = os.path.dirname(self.epitome_test_dir)
+        assert EpitomeDataset.get_data_dir(data_dir=epitome_test_data_dir, assembly="hg38") == os.path.join(epitome_test_data_dir, "hg38")
+
+        # Should error because undownloaded data path doesn't have a specified assembly
+        default_data_dir = os.path.join(DEFAULT_EPITOME_DATA_PATH, "hg19")
```

Comment:
but it doesnt error? this comment is confusing

Reply:
Fixed